tools/common/schema/setuptask.go

```diff
@@ -64,6 +64,7 @@ func (task *SetupTask) Run() error {
 		if err != nil {
 			return err
 		}
+		defer file.Close()
```

Comment:
good catch!
I'd recommend using defer only in smaller scope.
I suggest extracting the parsing to a separate function, then defer is OK.
Alternatively close right after ParseFile.